packages/cli/src/commands/repo/test.ts

```diff
@@ -54,26 +54,35 @@ interface GlobalWithCache extends Global {
  * Use git to get the HEAD tree hashes of each package in the project.
  */
 async function readPackageTreeHashes(graph: PackageGraph) {
-  const pkgs = Array.from(graph.values());
+  const pkgs = Array.from(graph.values()).map(pkg => ({
+    ...pkg,
+    path: relativePath(paths.targetRoot, pkg.dir),
+  }));
   const output = await runPlain(
     'git',
     'ls-tree',
-    '--object-only',
+    '--format="%(path)=%(objectname)"',
```

Comment:
Any file/dir with an equals sign breaks this right? Can we put the known-good thing first (the hash) and then split smartly on just the FIRST match of the separator?

Reply:
Yep, fix'd